apps/omg_utils/mix.exs

```diff
@@ -25,5 +25,13 @@ defmodule Utils.MixProject do
   defp elixirc_paths(:prod), do: ["lib"]
   defp elixirc_paths(_), do: ["lib", "test/support"]
 
-  defp deps, do: [{:statix, "~> 1.1"}, {:appsignal, "~> 1.9"}]
+  defp deps do
```

Comment:
maybe we should try keeping omg_utils thin and use omg_status for these things.

Reply:
I added these dependencies here because metrics currently live at `OMG.Utils.Metrics` 

After reading the `OMG.Status` README, however, it does seem like that's a better place for it. 😃 

> Status is a umbrella application. It's purpose is to gather and send metrics

Are you proposing that we move `OMG.Utils.Metrics` to `OMG.Status.Metrics` ? If so, would it make sense to do that in another PR since it will be quite noisy with all the `use OMG.Utils.Metrics` macros throughout the code?

Reply:
Yes. That should work (offloading to a separate pr) if it’s indeed part of this assignment/ticket.